LayoutTests/imported/w3c/web-platform-tests/cookie-store/cookieStore_event_delete.https.window-expected.txt

```diff
@@ -1,5 +1,3 @@
 
-Harness Error (TIMEOUT), message = null
-
-TIMEOUT cookieStore fires change event for cookie deleted by cookieStore.delete() Test timed out
+FAIL cookieStore fires change event for cookie deleted by cookieStore.delete() assert_equals: expected 1 but got 0
```

Comment:
When a cookie is deleted, it is deleted using the CookieStore::set() function, which corresponds to a cookie being ~changed~, not ~deleted~. So when the Cookie Store is a listener, the change function called id `cookiesAdded` and not `cookiesDeleted`. How should we fix this? One option is that in `cookiesAdded`, if the expiry of the cookie is more than 24 hours in the past (that's what we set the expiry to be when deleting a cookie by setting it), then we can call `cookiesDeleted` for that and other such cookies. Is there a different idea that could be better? Basically, `cookiesDeleted` isn't being called at all for CookieStore.

Reply:
Fixed this by checking if the expiry date of the cookie is in the past. If yes, it's added to the list of deleted cookies.